src/periphery/MgvOracle1559.sol

```diff
@@ -0,0 +1,72 @@
+// SPDX-License-Identifier:	AGPL-3.0
```

Comment:
This should be BUSL-1.1 which  `MgvOracle` and the other core contracts were also updated to recently.